src/app/pages/k12/subject/subject.scss

```
@@ -20,12 +20,20 @@
         }
 
         .items {
-            a {
-                padding: 0.5rem 2rem 0.2rem;
+            @include width-up-to($tablet-max) {
+                border-left: thin solid text-color(white);
```

A small suggestion, I think it's generally better to use precise units for readability and browser consistency. Assuming the root is `10px` (this may need to be tweaked):

```suggestion
                border-left: 0.1rem solid text-color(white);
```

---

I use thin for borders a lot throughout the app, and in general I'm ok with allowing the browser to supply its notion of "thin". Since this isn't functionally a border (it's more of a separator), I can go along with the more precise width without having to update all borders throughout the app for consistency. So you convinced me.